remote-web/src/styles.css

```
@@ -0,0 +1,954 @@
+  }
+
+  .quick-key-strip::-webkit-scrollbar {
+    display: none;
+  }
+
+  .quick-key-strip span {
+    position: absolute;
+    width: 1px;
+    height: 1px;
+    overflow: hidden;
+    clip: rect(0, 0, 0, 0);
```

Stylelint (17.9.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)